guides/common/modules/proc_adding-custom-rpm-repositories.adoc

```diff
@@ -1,57 +1,66 @@
-. From the *Download Policy* list, select the type of synchronization {ProjectServer} performs. See xref:Download_Policies_Overview[]
-. Ensure that the *Mirror on Sync* check box is selected.
+{Project} automatically completes the *Label* field based on what you have entered for *Name*.
+. Optional: In the *Description* field, enter a description for the repository.
+. From the *Type* list, select `yum` as type of repository.
+. Optional: From the *Restrict to Architecture* list, select an architecture.
+If you want to make the repository available to all hosts regardless of the architecture, ensure to select *No restriction*.
+. Optional: From the *Restrict to OS Version* list, select the OS version.
+If you want to make the repository available to all hosts regardless of the OS version, ensure to select *No restriction*.
+. In the *Upstream URL* field, enter the URL of the external repository to use as a source.
+* Optional: Check the *Ignore SRPMs* check box to exclude source RPM packages from being synchronized to {Project}.
+. Select the *Verify SSL* check box if you want to verify that the upstream repository's SSL certificates are signed by a trusted CA.
+. In the *Upstream Username* field, enter the user name for the upstream repository if required for authentication.
+Clear this field if the repository does not require authentication.
+. In the *Upstream Password* field, enter the corresponding password for the upstream repository.
+Clear this field if the repository does not require authentication.
+. From the *Download Policy* list, select the type of synchronization {ProjectServer} performs.
+For more information, see xref:Download_Policies_Overview[].
+. Ensure to select the *Mirror on Sync* check box.
 This ensures that the content that is no longer part of the upstream repository is removed during synchronization.
+. Optional: In the *HTTP Proxy Policy* field, select an HTTP proxy.
 . From the *Checksum* list, select the checksum type for the repository.
 . Optional: If you want, you can clear the *Publish via HTTP* check box to disable this repository from publishing through HTTP.
 . Optional: From the *GPG Key* list, select the GPG key for the product.
-. Click *Save*.
+. Optional: From the *SSL CA Cert* list, select a SSL CA certificate to verify the upstream repository's SSL certificates.
+. Optional: From the *SSL Client Cert* list, select an SSL client certificate to authenticate yourself against the external repository.
+. Optional: From the *SSL Client Key* list, select an SSL client key to authenticate yourself against the external repository.
+. Click *Save* to create the repository.
 
-[id="cli-adding-rpm-repositories"]
+[id="cli-adding-custom-rpm-repositories"]
 .CLI procedure
-
 . Enter the following command to create the repository:
 +
 [options="nowrap" subs="+quotes"]
 ----
 # hammer repository create \
---name "_My_Repository_" \
+--arch "_My_Architecture_" \
```

Comment:
Nitpick: `_My_Architecture_` is quite weird, we don't have that yet. I would probably go just with intel 64bits as that's what 90% of our users use. But fine with this too, it is ready for the (ARM) future :-)

Reply:
Technically, it is possible to serve content to non-x86 hosts. I feel like this goes along nicely with `My_Org` and `My_Loc`.